jest.config.js

```diff
@@ -26,7 +26,6 @@ module.exports = {
 	clearMocks: true,
 	preset: './gutenberg/node_modules/react-native/jest-preset.js',
 	setupFiles: [ '<rootDir>/' + configPath + '/setup.js' ],
-	testEnvironment: 'jsdom',
```

Comment:
Now we use `node` environment which is now the default value ([reference](https://jestjs.io/blog/2021/05/25/jest-27#flipping-defaults)).